examples/cc3200/xively_firmware_updates/SFT/xi_sft_client.c

```diff
@@ -1043,7 +1043,7 @@ void xi_parse_file_chunk( xi_context_handle_t in_context_handle,
 void verify_sha256( xi_context_handle_t in_context_handle )
 {
     /* Finalize SHA256 Digest */
-    xi_bsp_fwu_checksum_final( download_ctx.checksum_context,
+    xi_bsp_fwu_checksum_final( &download_ctx.checksum_context,
```

Comment:
is this due to the upcoming HTTP changes?

Reply:
no. The checksum context deallocation is the responsibility of the `...final` function. And there is a style in the library that the deallocator does the NULL-ing the pointer itself. And this caused problem during destruction of SFT context: the checksum context was re-deallocated :D because the pointer wasn't nullified at the "normal" deallocation.